src/issuance.rs

```diff
@@ -1305,7 +1301,7 @@ mod tests {
     fn issue_bundle_verify_fail_wrong_asset_descr_size() {
         // we want to inject "bad" description for test purposes.
         impl IssueAction {
-            pub fn modify_descr(&mut self, new_descr: String) {
+            pub fn modify_descr(&mut self, new_descr: Vec<u8>) {
```

Comment:
please modify comment to
`We want to inject a "malformed" description for test purposes.`

Reply:
Done